acls/docker-compose.yaml

```diff
@@ -0,0 +1,81 @@
+---
+version: '3'
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:5.3.0
+    hostname: zookeeper
+    container_name: zookeeper
+    environment:
+      ZOOKEEPER_SERVER_ID: 1
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: "2000"
+      KAFKA_JMX_PORT: 9999
+      KAFKA_JMX_HOSTNAME: localhost
+      KAFKA_OPTS: -Djava.security.auth.login.config=/tmp/zookeeper.sasl.jaas.conf 
+           -Dzookeeper.authProvider.1=org.apache.zookeeper.server.auth.SASLAuthenticationProvider
+    ports:
+      - 2181:2181
```

Comment:
Any reason to bind this port? it makes the deployment fail on my laptop as I have a local ZooKeeper:

`ERROR: for zookeeper  Cannot start service zookeeper: driver failed programming external connectivity on endpoint zookeeper (3c557dd57f2031fd919c91e7936b6f1dfccbcf9109fdf82ec4062a0019a2c95f): Error starting userland proxy: listen tcp 0.0.0.0:2181: bind: address already in use`


```suggestion
```